app/src/androidTest/java/com/google/android/perftesting/testrules/EnableBatteryStatsDump.java

```diff
@@ -73,6 +73,18 @@ public Statement apply(Statement base, Description description) {
 
     @Override
     public void before() {
+        begin();
+    }
+
+    public void after() {
+        File file = new File(String.valueOf(getTestFile(mTestClass, mTestName,
```

Comment:
Duplicate code. There must be another way to check if battery stats has been collected.